### PR TITLE
Removed bootstrap javascript

### PR DESCRIPTION
**Pull request by Nyholm**
The functionality is the same. We dont use jquery or any bootstrap js.

| File                  | Before   | After
| ------------- | -------- | ------------------|
| vendor.js        | 121 KB   | 26 KB
| vendor.css     | 73 KB   | 69 KB
| app.js              | 4 KB   | 4 KB
| app.css           | 4 KB   | 5 KB

### CODE REVIEW

File: website/assets/script/sidebar.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+let isUlHidden = function (ul) {
+    return !ul.classList.contains('show');
+};
+
+
+// The submenu
+let elements = document.getElementsByClassName('dropdown-toggle');
+for (let i = 0; i < elements.length; i++) {
+    elements[i].addEventListener("click", function(event){
+        let link = el = elements[i];
```
Comment (jderusse) [inline website/assets/script/sidebar.js:16]: const link

File: website/assets/script/sidebar.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+});
+
+let isUlHidden = function (ul) {
+    return !ul.classList.contains('show');
+};
+
+
+// The submenu
+let elements = document.getElementsByClassName('dropdown-toggle');
+for (let i = 0; i < elements.length; i++) {
```
Comment (jderusse) [inline website/assets/script/sidebar.js:14]: for const element of elements

File: website/assets/script/sidebar.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    elements[i].addEventListener("click", function(event){
```
Comment (jderusse) [inline website/assets/script/sidebar.js:15]: event is not used. and can be replace by arrow function
() => {

File: website/assets/script/sidebar.js (hunk 2 above)
Comment (Nyholm) [reply website/assets/script/sidebar.js:14]: I did not know you could write a loop like that.. =)

**Comment by Nyholm** [comment]
PR is updated

**Pull request merged**